Docs import paths (#3087)
GeotrekCE/Geotrek-admin: docs/install/import.rst

Import data
===========

Import paths
============

.. danger::

>     With dynamic segmentation, importing paths is only possible if no paths are created yet.

Reviewer: ```suggestion
    With dynamic segmentation, importing paths is only safe if no paths are created yet.
```

Author: J'ai un peu modifié la phrase

>     If you import paths where there are existing paths or treks, their linked topology will be modified automatically.

Reviewer: ```suggestion
    If you import paths where there are existing paths or treks, their linked topology might be impacted.
```

Author: J'ai un peu modifié la phrase

Import data
===========

Import paths
============

.. danger::

>     With dynamic segmentation, importing paths is very risky if paths are already present in Geotrek,

Reviewer 2: I would add "in the same area". Because importing some paths in an area without pre-existing ones, is not a problem. 
I also import paths all the time in areas that already have paths, I just take care in QGIS to msnap new paths with pre-existing ones. 
I use this method : https://makina-corpus.com/sig-webmapping/importer-une-couche-de-troncons-dans-geotrek

So I may wouldn't be so scary. 
But maybe with the UI import tool, it is more complicated and more risky. 

The good way to add paths in a Geotrek with existing one, is to prepare them in QGIS with displaying the path layer from Geotrek to snap new paths to existing ones, and to never import a path overriding an existing one.

Author: Ah yes good point, and I've forgottent this article from Mathieu :)

Author: Hello, I've made some improvements with your feedback

>     it is only safe with no path created.

Reviewer: ```suggestion
    it is only safe with no pre-existing paths.
```

    Indeed, if you import paths where there are existing paths, treks, POIs or trails linked topology might be impacted.

> Before import paths layer, it is important to prepare them. Paths must be:

Reviewer: ```suggestion
Before importing paths layer, it is important to prepare them. Paths must be:
```
